toxcore/network.h

```diff
@@ -102,34 +102,39 @@ typedef int sock_t;
 
 #define MAX_UDP_PACKET_SIZE 2048
 
-#define NET_PACKET_PING_REQUEST    0   /* Ping request packet ID. */
-#define NET_PACKET_PING_RESPONSE   1   /* Ping response packet ID. */
-#define NET_PACKET_GET_NODES       2   /* Get nodes request packet ID. */
-#define NET_PACKET_SEND_NODES_IPV6 4   /* Send nodes response packet ID for other addresses. */
-#define NET_PACKET_COOKIE_REQUEST  24  /* Cookie request packet */
-#define NET_PACKET_COOKIE_RESPONSE 25  /* Cookie response packet */
-#define NET_PACKET_CRYPTO_HS       26  /* Crypto handshake packet */
-#define NET_PACKET_CRYPTO_DATA     27  /* Crypto data packet */
-#define NET_PACKET_CRYPTO          32  /* Encrypted data packet ID. */
-#define NET_PACKET_LAN_DISCOVERY   33  /* LAN discovery packet ID. */
-
-/* See:  docs/Prevent_Tracking.txt and onion.{c, h} */
-#define NET_PACKET_ONION_SEND_INITIAL 128
-#define NET_PACKET_ONION_SEND_1 129
-#define NET_PACKET_ONION_SEND_2 130
-
-#define NET_PACKET_ANNOUNCE_REQUEST 131
-#define NET_PACKET_ANNOUNCE_RESPONSE 132
-#define NET_PACKET_ONION_DATA_REQUEST 133
-#define NET_PACKET_ONION_DATA_RESPONSE 134
-
-#define NET_PACKET_ONION_RECV_3 140
-#define NET_PACKET_ONION_RECV_2 141
-#define NET_PACKET_ONION_RECV_1 142
-
-/* Only used for bootstrap nodes */
-#define BOOTSTRAP_INFO_PACKET_ID 240
-
+/* MUST NEVER EXCEED UINT8_MAX */
+typedef enum NET_PACKET_TYPE {
+    NET_PACKET_PING_REQUEST     = 0,   /* Ping request packet ID. */
+    NET_PACKET_PING_RESPONSE    = 1,   /* Ping response packet ID. */
+    NET_PACKET_GET_NODES        = 2,   /* Get nodes request packet ID. */
+    NET_PACKET_SEND_NODES_IPV6  = 4,   /* Send nodes response packet ID for other addresses. */
+    NET_PACKET_COOKIE_REQUEST   = 24,  /* Cookie request packet */
+    NET_PACKET_COOKIE_RESPONSE  = 25,  /* Cookie response packet */
+    NET_PACKET_CRYPTO_HS        = 26,  /* Crypto handshake packet */
+    NET_PACKET_CRYPTO_DATA      = 27,  /* Crypto data packet */
+    NET_PACKET_CRYPTO           = 32,  /* Encrypted data packet ID. */
+    NET_PACKET_LAN_DISCOVERY    = 33,  /* LAN discovery packet ID. */
+
+    NET_PACKET_DATA_REQUEST     = 50,
+    NET_PACKET_DATA_RESPONSE    = 51,
+
+    /* See:  docs/Prevent_Tracking.txt and onion.{c, h} */
+    NET_PACKET_ONION_SEND_INITIAL   = 128,
+    NET_PACKET_ONION_SEND_1         = 129,
+    NET_PACKET_ONION_SEND_2         = 130,
+
+    NET_PACKET_ANNOUNCE_REQUEST     = 131,
+    NET_PACKET_ANNOUNCE_RESPONSE    = 132,
+    NET_PACKET_ONION_DATA_REQUEST   = 133,
+    NET_PACKET_ONION_DATA_RESPONSE  = 134,
+
+    NET_PACKET_ONION_RECV_3 = 140,
+    NET_PACKET_ONION_RECV_2 = 141,
+    NET_PACKET_ONION_RECV_1 = 142,
+
+    /* Only used for bootstrap nodes */
+    BOOTSTRAP_INFO_PACKET_ID = 240,
```

Comment:
Perhaps add a `NET_PACKET_TYPE_MAX = 255` and put that "Must never exceed" comment on it. Also, no need to shout at the reader.

Also, this change is unrelated to tox-named, but makes sense. Can you make a separate PR with just this change and reasoning about why this change is good?